Arc/Arc.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Agate styled arc components and behaviors.
+ *
+ * @example
+ * <Arc color="blue" endAngle={200} startAngle={0}" radius={100} />
+ *
+ * @module agate/Arc
+ * @exports Arc
+ * @exports ArcBase
+ * @exports ArcDecorator
+ */
+
+import kind from '@enact/core/kind';
+import Pure from '@enact/ui/internal/Pure';
+import ri from '@enact/ui/resolution';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+import React from 'react';
+import Skinnable from '../Skinnable';
+
+import {arcPath} from './utils';
+
+/**
+ * An arc component.
+ *
+ * This component is most often not used directly but may be composed within another component as it
+ * is within [Arc]{@link agate/Arc.Arc}.
+ *
+ * @class ArcBase
+ * @memberof agate/Arc
+ * @ui
+ * @public
+ */
+const ArcBase = kind({
+	name: 'Arc',
+
+	propTypes: /** @lends agate/Arc.ArcBase.prototype */ {
+		/**
+		 * The color of the arc.
+		 *
+		 * @type {String}
+		 * @default: #000000
+		 * @public
+		 */
+		color: PropTypes.string,
+
+		/**
+		 * The end angle(in degrees) of the arc.
+		 *
+		 * The value should be between 0 and 360 and should be greater than startAngle.
+		 *
+		 * @type {number}
+		 * @default: 310
+		 * @public
+		 */
+		endAngle: PropTypes.number,
+
+		/**
+		 * The radius of the arc.
+		 *
+		 * @type {number}
+		 * @default: 100
+		 * @public
+		 */
+		radius: PropTypes.number,
+
+		/**
+		 * The start angle(in degrees) of the arc.
+		 *
+		 * The value should be between 0 and 360.
+		 *
+		 * @type {number}
+		 * @default: 50
+		 * @public
+		 */
+		startAngle: PropTypes.number,
+
+		/**
+		 * The stroke width of the arc.
+		 *
+		 * @type {number}
+		 * @default: 1
+		 * @public
+		 */
+		strokeWidth: PropTypes.number
+	},
+
+	defaultProps: {
+		color: '#000000',
+		endAngle: 310,
+		radius: 100,
+		startAngle: 50,
+		strokeWidth: 1
```

Comment:
Is this right? should it be 2?

Reply:
it is only a default value. I can put 2 instead  :)

Reply:
Oh, I think we could set this as `9` so it looks similar to the GUI guide.